src/lib/manager_impl.cpp

```diff
@@ -44,6 +41,12 @@ class engine_factory : public group_factory {
                                          start_params.ssl_cert_),
             application_(app) {}
 
```

Comment:
Is the constructor above used anymore?

Reply:
No, I wanted to ensure that we will never go back to the common threadpool before removing it completely.